docs/Load_Cell.md

```diff
@@ -120,3 +124,366 @@ a macro:
 ```
 {% set tare_counts = printer.load_cell.tare_counts %}
 ```
+
+# Load Cell Probes
+
+## Related Documentation
+
+* [load_cell_probe Config Reference](Config_Reference.md#load_cell_probe)
+* [load_cell_probe G-Code Commands](G-Codes.md#load_cell_probe)
+* [load_cell_probe Statuc Reference](Status_Reference.md#load_cell_probe)
+
+## Load Cell Probe Safety
+
+Because load cells are a direct nozzle contact probe there is a risk of
+damage to your printer if too much force is used. The load cell probing system
+includes a number of safety checks that try to keep your machine safe from
+excessive force to the toolhead. It's important to understand what they are
+and how they work as you can defeat most of them with poorly chosen config
+values.
+
+#### Calibration Check
+Every time a homing move starts, load_cell_probe checks
+that the load_cell is calibrated. If not it will stop the move with an error:
+`!! Load Cell not calibrated`.
+
+#### `counts_per_gram`
+This setting is used to convert raw sensor counts into grams. All the safety
+limits are in gram units for your convenience. If the `counts_per_gram`
+setting is not accurate you can easily exceed the safe force on the toolhead.
+You should never guess this value. Use `LOAD_CELL_CALIBRATE` to find your load
+cells actual `counts_per_gram`.
+
+#### `trigger_force`
+This is the force in grams that triggers the endstop to halt the homing move.
```

Comment:
The removal of endstop functionality has left the docs in a slightly confusing state. I'd be happy to take a stab at a PR to clean it up, but I'm not familiar enough with the code to get all the details right. For example, here I'm not clear whether the wording would correctly be "homing move" or not. @garethky would you review for accuracy if I gave an edit a go?

Reply:
Oh my goodness do I need an editor. Yes!

Technically, it is making a "homing move". This is a move that is terminated by a sensor and it's using the homing module to do it.

But it is not homing an axis, which is the act of taking some position and assigning it to the current position of the axis.

I doubt anyone reading this is going to care about that hair splitting.

Reply:
Thanks. To me that seems like an implementation detail of probing, so I think it'd be best to avoid saying "homing" in the docs (other than where it currently explains how to use homing override to achieve homing). I'll tag you when I put it up :)